.github/workflows/pre-release-internal-env.yml

```diff
@@ -50,15 +53,29 @@ jobs:
         id: find-versions
         run: |
           version=$(yq ".version" helm/fiftyone-teams-app/Chart.yaml)
-          if [[ "${GITHUB_REF_NAME}" == "release/"* ]]; then
-            version="${version}-rc-${GITHUB_SHA::7}"
+          sha="$(git rev-parse --short HEAD)"
+
+          if [[ "${{ github.event_name }}" == "pull_request" ]]; then
+            # If this was triggered via a pull_request and then workflow_call.
+            # The workflow is triggered by pull_request, not workflow_call,
+            # which is why the above conditional uses the event name pull_request.
+            # In this case, it is a development branch, so mark it as a
+            # sha build.
+            version="${version}-sha-${sha}"
+          elif [[ "${{ github.event_name }}" == "push" ]]; then
+            # After the PR is merged, a push event is triggered to
+            # the branch. So mark it as an rc build.
+            version="${version}-rc-${sha}"
           else
-            version="${version}-sha-${GITHUB_SHA::7}"
+            # This isn't really needed but, in my opinion, adds clarity
+            # to the above two conditionals.
+            echo "This was triggered by ${{ github.event.name }}. Exiting..."
```

Comment:
I like the clarity of this; if we do get an unexpected error it'll tell us what the event was.  Thank you.